nncf/quantization/algorithms/hyperparameter_tuner/param_grid.py

```diff
@@ -75,6 +76,12 @@ def _get_minmax_quantization_param_grid() -> ParamGrid:
             RangeEstimatorParameters(min=min_v, max=max_v)
             for min_v, max_v in itertools.product(min_param_values, max_param_values)
         ],
+        "advanced_parameters:quantizer_propagation_rule": [
+            QuantizerPropagationRule.DO_NOT_MERGE_BRANCHES,
```

Comment:
What is reason to check 4 options?

Reply:
Rollbacked to 2 combinations. I checked out the number of combinations with these changes is increased from 180 to 540.

Reply:
Upd: The number of combinations is not equal to the number of considered combinations by a tuner. 
Tuner creates all possible combinations but considers only a few of them.

After these changes: at first, the tuner starts to find the best configuration for `quantizer_propagation_rule` by choosing from `None` and `QuantizerPropagationRule.MERGE_IF_ALL_BRANCHES_SAME`. Then, choosing the best value, the tuner considers the next parameters.
Thus, it leads to only 1 extra configuration to consider.